src/core/shelf.sol

```diff
@@ -52,12 +52,13 @@ contract Shelf is TitleOwned {
     struct Loan {
         address registry;
         uint256 tokenId;
-        uint price; //
-        uint principal; //
-        uint initial; //
+        uint price;
+        uint principal;
+        uint initial;
     }
 
     mapping (uint => Loan) public shelf;
+    mapping (uint => uint) public locks;
```

Comment:
Why not `bool` why not inside the loans struct?

Reply:
For this reason: https://gist.github.com/lucasvo/5cf682c29325e450b12b4f963c856cf1